bin/pgenv

```diff
@@ -935,6 +937,22 @@ pgenv_start_instance(){
         echo "PostgreSQL $v started"
         echo "Logging to $PGENV_LOG"
 
+	# if this is the first start, i.e., the start run
+	# just after the initialization, execute
+	# the appropriate custom script
+	if [ $PGENV_INITDB_DONE -eq 1 ]; then
+	    if [ ! -z "$PGENV_SCRIPT_FIRSTSTART" -a -x "$PGENV_SCRIPT_FIRSTSTART" ]; then
+		cat <<EOF
+This is the first time PostgreSQL $v is started,
+executing the \`PGENV_SCRIPT_FIRSTSTART\` script
+   $PGENV_SCRIPT_FIRSTSTART"
```

Comment:
No opening quotation mark here. Should it be this?

```suggestion
   "$PGENV_SCRIPT_FIRSTSTART"
```

Reply:
FIXED